oe_theme.theme

```diff
@@ -213,9 +213,11 @@ function oe_theme_theme_suggestions_paragraph_alter(array &$suggestions, array $
 function oe_theme_preprocess_paragraph__oe_links_block(array &$variables): void {
   // Massage data to be compliant with ECL links block component data structure.
   foreach (Element::children($variables['content']['field_oe_links']) as $index) {
+    $url = $variables['content']['field_oe_links'][$index]['#url'];
     $variables['links'][] = [
       'label' => $variables['content']['field_oe_links'][$index]['#title'],
-      'url' => $variables['content']['field_oe_links'][$index]['#url'],
+      'url' => $url,
+      'is_external' => ($url->isExternal() && !str_contains($url->getUri(), 'europa.eu')),
```

Comment:
europa.eu could be present in another section of the url as well, I think it's better to parse the url and check for the path, so we are sure that the domain is what we check against.